music/webpack/es5.base.config.ts

```diff
@@ -6,7 +6,7 @@ export const baseConfig = {
     rules: [{
       test: /\.ts$/,
       exclude: /node_modules/,
-      use: {loader: 'ts-loader', options: {configFile: 'tsconfig.es5.json'}}
+      use: {loader: 'ts-loader', options: {configFile: 'tsconfig.es6.json'}}
```

Comment:
I don't think this is correct. This is the es5 config file, and should be using the es5 rules, no?

Reply:
upgrading tonejs (which now is in typescript) started to give errors and some googling gave me this result:
https://stackoverflow.com/questions/50203369/class-constructor-cannot-be-invoked-without-new-typescript-with-commonjs/50203532#50203532 specifically this: ```This problem should be solved in Node.js by setting TypeScript target option to es6 or higher. Modern Node.js versions support ES6 classes, there is no need to transpile them.
```

Happy to change this back to a working es5 tsconfig but I'll require some help from you to get it right.

Reply:
Hmm, i don't think the node build should be using the es5.base.config. I am worried that if we're changing it here, then our es5 build will not be an es5 build anymore.

Reply:
thanks for catching this. 

i changed this back to the es5 config and for some reason, I'm no longer running into the issues that I was having before! it's very perplexing, so I went through all the demoes to make sure everything still works and... they do. perhaps the issue I was having had to do with something else which I fixed.